fe/fe-core/src/main/java/com/starrocks/analysis/AnalyticWindow.java

```diff
@@ -376,6 +381,11 @@ public boolean equals(Object obj) {
                 && rightBoundaryEqual;
     }
 
+    @Override
+    public int hashCode() {
+        return Objects.hash(type_, leftBoundary_, rightBoundary_, toSqlString_);
```

Comment:
need toSqlString_?  euqal method no toSqlString_.

Reply:
sry, haven't notice that. I'll remove this field